File: NARESH_it/1_C_class/1_C_ENcryption_lab/DAYS 2 29-AUG-24/p_2.c
/**
 * @Author: Dastan Alam
 * @Date:   2024-08-29 06:21:19 PM   18:08
 * @Last Modified by:   Dastan Alam
 * @Last Modified time: 2024-08-29 06:24:20 PM   18:08
 */

// Que 2 :  Write a 'C' program to find next multiple of 10 of given 2 digit number only.
// --------

// Sample input : int a = 34;

// Sample Output : 40
#include <stdio.h>
#include <stdlib.h>
#include <stdint.h>
#include <string.h>


int main()
{
    int a;
    printf("enter the number ");
    scanf("%d",&a);
    // printf("%d",a/10);
    printf("%d",((a/10+1)*10));
    return 0;
}

File: NARESH_it/3_java/DAY 5 28-OCT-24/p_2.java
/*
Que 2 :
=======

Write a java program to find the ASCII value of a character
For Example
Output sholud be in below formart
The ASCII value of a is : 97
=================================================
*/
import java.util.Scanner;
class p_2 
{
	public static void main(String[] args) 
	{
		Scanner sc=new Scanner(System.in);
		System.out.print("enter the char : ");
		char x=sc.next().charAt(0);
		int y=x;
		System.out.println("The ASCII value of a is : "+y);
	}
}
